android/app/src/main/res/xml/backup_content.xml

```diff
@@ -2,4 +2,4 @@
 <full-backup-content>
   <include domain="file" path="bookmarks/"/>
   <include domain="file" path="settings.ini"/>
-</full-backup-content>
+</full-backup-content>
```

Comment:
ditto

Reply:
@biodranik  There were extra empty lines at the end of the code in my IDE when compared to the original Organicmaps repo, so I removed those line thinking those changes were made by my IDE.

Reply:
https://www.merriam-webster.com/dictionary/ditto